### ISSUE TITLE
Admin language-loading: No means for a site to override a `zc_plugin` extra_definitions file

### ISSUE THREAD

**Issue by lat9** [zencart/zencart#6537]
Yet another corollary to [this](https://github.com/zencart/zencart/pull/6493) storefront PR, there's no way for a site to _easily_ override any  `admin/includes/languages/english/extra_definitions` files provided by a `zc_plugin`.

To easily see the issue, create a language file (`admin/includes/languages/english/extra_definitions/lang.override.php`) containing
```php
<?php
$define = [
      'BOX_TOOLS_DISPLAY_LOGS' => 'Display Log Files (ed)',
];
return $define;
```

My expectation is to see the admin's `Tools` menu now show an element named `Display Log Files (ed)` instead of the base `Display Log Files`, as provided by the `zc_plugin`.

The challenge in the admin side is that there's no template with which to provide that override.  As such, changing the load order of the `extra_definitions` so that a `zc_plugin` language constant can be overridden has the side-effect of no longer allowing a `zc_plugin` to override a 'base' extra_definition constant.

**Comment by piloujp** [comment]
Putting the file in `..\zc_plugins\DisplayLogs\v3.0.2\admin\includes\languages\english\extra_definitions` will do what you want.
After all, it is the plugin you want to override.

**Comment by lat9** [comment]
True, but then that change will be lost if/when an update to the `zc_plugin` is provided.

**Comment by drbyte** [comment]
Aside: language translations can't be added for plugins without manually adding them to a plugin's subdirectory tree, vs directly in the non-zc plugins core directory.
And no fallback to English either, so plugins throw fatal errors when adding a translation pack if a zc plugin doesn't have lang files for that lang.

**Comment by lat9** [comment]
Noting that, for non-english languages, PR #6706 will enable those language translations to be specified in the 'root' admin language extra_definitions directory, e.g. `/admin/includes/languages/spanish/extra_definitions/lang.display_logs.php`, _so long as_ the encapsulated plugin doesn't also use an extra_definitions file for its page-specific language file (which it _shouldn't_).

That PR also corrects the fatal error when a plugin has only english translations in a non-english store.
